src/main/java/com/example/htmlpractice/model/ZipcodeFrom.java

```diff
@@ -0,0 +1,20 @@
+package com.example.htmlpractice.model;
+
+import jakarta.validation.constraints.Pattern;
+import lombok.Data;
+import org.hibernate.validator.constraints.Length;
+
+@Data
+public class ZipcodeFrom {
+    @Length(min = 3, max = 7, message = "3から７文字で入力してください")
```

Comment:
たとえばFormに100を入れたときにどういう挙動をしますか？
郵便番号検索の結果が表示されますか？

Reply:
あと、0000000のように7桁だけど存在しない郵便番号の場合どういう挙動になるか確認してほしいです。

Reply:
それぞれjava.lang.NullPointerException: Cannot invoke "java.util.Collection.toArray()" because "c" is nullと同じエラーが出てしまう状態です。複数データが返却される方はプログラムを書き換えて表示できるようにする予定です。
存在しないデータの場合は例外処理かなとぼんやり考えている状態です。
少し考えてみます。

Reply:
> たとえばFormに100を入れたときにどういう挙動をしますか？　  

外部APIから返ってきたメッセージで "パラメータ「郵便番号」の桁数が不正です。"と表示されていたため、外部APIの仕様書を読み返したところ7桁のみしか対応していないことに気づき以下2点変更しました。  
* 外部APIからのメッセージも表示するようにHTMLを変更(cc0b73c8cda756d9ebd206f7c16db7636db04bbd)
* 7桁のみ入力するようにバリデーション変更(85fe6108643035ef796523516b574e8456bd4a68)

> あと、0000000のように7桁だけど存在しない郵便番号の場合どういう挙動になるか確認してほしいです。

nullpointerexceptionが生じていました。address.getResults()でヌルポとなっていたためif分岐を追加しました(95f6da647aa9c771c07d057e950549a20f942f8c)
＊複数住所が返ってくる場合の動作確認がまだだったので一緒に載せておきます。
## 動作確認結果
### 外部APIのメッセージ表示確認 
<img  src="https://github.com/kamide28/findaddress/assets/127748178/dcd976de-728e-45f9-b4a3-513f4724d8aa" width=60%>

### バリテーションにて７文字のみに(Formに100を入力)
<img src="https://github.com/kamide28/findaddress/assets/127748178/7a5581bd-aecb-4937-b056-ce194603eaa0" width=60%>

### 0000000を入力(存在しない郵便番号)
<img src="https://github.com/kamide28/findaddress/assets/127748178/e8d8cd6b-fe39-4d3b-bd98-ac4ffaf34015" width=60%>

### 7桁で複数箇所住所が当てはまるとき
<img  src="https://github.com/kamide28/findaddress/assets/127748178/c99ea3f9-d28d-400a-9a9c-dc296a8378b4" width=60%>